### PR TITLE
adding shadow to features items

### LINKED ISSUE #48: adding shadow to features items
It will look awesome.

## Preview 

shadow

@nicolas-raoul should I go for this changes. one more change that I want is make adding little transformation on hovering on features images.

### PR DESCRIPTION
**Pull request by Vishal-raj-1**
## Before

f3

## Now

features

@nicolas-raoul Please review it !! This will fixes #48 .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,6 +275,8 @@ <h2>Commons Mobile App</h2>
         <div class="qr-code-div">
             <img src="images/commons-app-qr.png"/>
         </div>
+        
+        <a href="https://github.com/commons-app" class="btn btn-primary">View Commons App at GitHub</a>
```
Comment (nicolas-raoul) [inline index.html:279]: This change is unrelated, right?

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Vishal-raj-1) [reply index.html]: Fixed

**Comment by nicolas-raoul** [comment]
Thanks for fixing!

**Comment by Vishal-raj-1** [comment]
Thanks for merging 😊
